modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/FragmentEntryFragmentRenderer.java

```diff
@@ -347,6 +367,28 @@ private String _renderFragmentEntryLink(
 		return content;
 	}
 
+	private String _renderReactFragmentEntryLink(
+			long fragmentEntryLinkId, HttpServletRequest httpServletRequest)
+		throws PortalException {
+
+		Writer writer = new CharArrayWriter();
+
+		try {
+			_reactRenderer.renderReact(
+				new ComponentDescriptor(
+					ModuleNameUtil.getModuleResolvedId(
+						_jsPackage, "fragmentEntryLink/" + fragmentEntryLinkId),
+					"fragment" + fragmentEntryLinkId, Collections.emptyList(),
```

Comment:
If I understand this properly... in the end we ended up just replacing the `react` dependency when registering the fragment jsModule rather than creating an intermediate component somewhere inside DXP that could resolve this properly?

Reply:
What do you mean with "intermediate component somewhere inside DXP that could resolve this properly"?

Reply:
Resolve what?

Reply:
Okay, we worked this out offline, all clear... we're simply assuming `react-web` is provided and that's it.